src/client/package.json

```
@@ -9,17 +9,17 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "next": "14.2.15",
+    "next": "15.0.3",
     "node-fetch": "^3.3.2",
     "react": "^18",
     "react-dom": "^18"
   },
   "devDependencies": {
-    "@types/node": "^20",
+    "@types/node": "^2",
```

blocking: this should be v20 (unchanged)
```suggestion
    "@types/node": "^20",
```